js/menu.js

```diff
@@ -0,0 +1,40 @@
+const btn = document.querySelector('.mb-menu');
+const menu = document.querySelector('.hamburger');
+const menuList = document.querySelectorAll('.mb-menu-list a');
+
+btn.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  document.body.classList.toggle('no-scroll');
```

Comment:
- [ ] It looks that the issue with the scrolling is still giving some problems here. If you open your developer tools and check the inspector when you click on the hamburger menu button, the class `no-scroll` is added (as intended) but you didn't set any rule to remove that class after clicking somewhere else. The only way to remove the class is by clicking on the menu button again and in order to do that you have to either close the menu button and then open it again or either click a link and open again the menu. Please take a look at this issue and try to apply the necessary changes.